### PR TITLE
[windows][system-probe] Read from driver and set TCPEstablished/Closed on ConnectionStats object

### PR DESCRIPTION
**Pull request by brycekahle**
### What does this PR do?

Read the established/closed flags on flows from the windows driver, so we have basic churn metrics for Windows. Pairs with https://github.com/DataDog/datadog-windows-filter/pull/61

### Motivation

This improves the Windows system-probe metric offering.

### Describe your test plan

Test using the windows system-probe testing infrastructure.

### CODE REVIEW

File: pkg/network/ddnpmapi.h [hunk 1]
```diff
@@ -122,12 +122,13 @@ typedef struct _filterAddress
...
     uint8_t                   v4_padding[4];    // pad out to 64 bit boundary
     uint8_t                   v6_address[16];
-    uint64_t                  mask; // number of mask bits.  
+    uint64_t                  mask; // number of mask bits.
 } FILTER_ADDRESS;
 
 #define     DIRECTION_INBOUND    ((uint64_t)0)
 #define     DIRECTION_OUTBOUND   ((uint64_t)1)
 
+//#define     FILTER_LAYER_IPPACKET       ((uint64_t) 0)
```
Comment (stb2132) [inline pkg/network/ddnpmapi.h:131]: Is there a reason this is commented out/should it be removed in general?

Comment (brycekahle) [reply pkg/network/ddnpmapi.h:131]: I copy/pasted from the driver to make sure it was identical.

Comment (brycekahle) [reply pkg/network/ddnpmapi.h:131]: https://github.com/DataDog/datadog-windows-filter/blob/d0c8ded189ec4dafb28612311d13aa63948b3425/include/ddnpmapi.h#L131

Comment (stb2132) [reply pkg/network/ddnpmapi.h:131]: Makes sense! LGTM to me then.

**Review by stb2132 — APPROVED** [review]

**Review by derekwbrown — APPROVED** [review]

**Pull request merged**